README.md

```diff
@@ -91,6 +91,18 @@ $query = QueryBuilder::create('INBOX.Sent')
 $emails = $client->getEmails($query);
 ```
 
+Both `getEmails()` and `getEmailIds()` can take a third optional `boolean` argument. it can be set to false to indicate change of flag for fetched emails to seen.
+
+```php
+$query = QueryBuilder::create('INBOX.Sent')
+    ->youngerThan(3600)
+    ->flagSeen(true)  
+                     
+    ->getQuery();
+
+$emails = $client->getEmails($query,'INBOX', false); // set to "false" fetched emails' state will be set to seen. "true" no change of state will occur. default is true
```

Comment:
Missing a space between `$query,'INBOX'`

Also could you put the comment at the start of the line above? That will be easier to read, else there will be a scrollbar on GitHub.